doc/k8s/README.md

```diff
@@ -0,0 +1,53 @@
+# Running SQLFlow on Kubernetes
+
+This is a tutorial on how to run SQLFlow on Kubernetes, this tutorial will deploy:
+- MySQL server with the SQLFlow example datasets loaded, and
+- SQLFlow component contains the sqlflow-server the Jupyter notebook.
+
+Then you can run the SQLFlow query in the Jupyter notebook on the browser.
+
+## Preparation
+
+1. Kubernetes cluster: You can check the [offical page](https://kubernetes.io/docs/setup/) to set up a
```

Comment:
Kubernetes cluster => Setup a Kubernetes cluster
You can check the ... => You can refer to the official page to set up a full cluster or use a local quick start tool: minikube. In this tutorial .....

Reply:
done, thx.